app/grpc/Report.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
```

Comment:
Can we move these proto files to the protobuf repo please, so that we can have one repo for all protobuf messages? What say?

Reply:
yes, that is what I wanted to do initially, but had some issues. I replied you in skype. Thanks

Reply:
Moving this task into our backlog